src/learn/augur-to-auspice.rst

```diff
@@ -391,8 +391,13 @@ The colorings in the auspice-config JSON typically link the key (where the
 metadata is to be found) to a title (how the key is to be presented in Auspice)
 and a type (how should the color scale be constructed).
 Setting the type to one of “continuous”, “temporal”, “ordinal”, “categorical”
-or “boolean” lets Auspice use the appropriate color scale, but if this is not
-provided “categorical” is the default.
+or “boolean” lets Auspice use the appropriate color scale.
+If the type is not provided, Augur will try to guess the appropriate type based
+on values in the field:
+
+#. "boolean" if values only consist of ``["false", "true", "1.0", "0.0", "1", "0", "yes", "no"]``
+#. "continuous" if values are all numeric
+#. "categorical" is the default
```

Comment:
Alternate wording:

> If the type is not provided, it will be inferred in this order:
> 
> 1. "boolean" if all values are within `["false", "true", "1.0", "0.0", "1", "0", "yes", "no"]`.
> 2. "continuous" if all values are numeric.
> 3. "categorical" if none of the above are satisfied.
>

Reply:
Thanks! Updated in force-push.